@here/harp-vectortile-datasource/lib/adapters/geojson-vt/GeoJsonVtDataAdapter.ts

```diff
@@ -181,20 +181,29 @@ export class GeoJsonVtDataAdapter implements DataAdapter {
                     break;
                 }
                 case VTJsonGeometryType.Polygon: {
-                    const polygon: IPolygonGeometry = { rings: [] };
+                    const polygons: IPolygonGeometry[] = [];
```

Comment:
Could you add a test in GeoJsonVtAdapterTest to check that nested rings are separated into polygons?

Reply:
done